source/YTEPs/YTEP-0037.rst

```diff
@@ -196,20 +204,24 @@ Enforcing these change throughout future contributions can be done by
 
 It is expected that transitioning to the "blackened" version of the code will add a bit
 of overhead in merging pre-existing PRs. Specifically, a simple ``git merge <pr-branch>
-<target-blackened-branch>`` will almost certainly raise git conflicts. A potential
-solution to this is to sanitize the pr-branch (on author side) with:
+master`` will almost certainly raise git conflicts. One possible solution to this is to
+sanitize the pr-branch (on author side) with:
 
 .. code:: shell
 
     pip install black
     black --line-width <N> yt/
```

Comment:
I was confused by this instruction because I thought we had chosen a specific line width of 88 characters. Should we tell the user to go with that?

Reply:
Oh also, it should be `--line-length`.

Reply:
>  I thought we had chosen a specific line width of 88 characters. 

we did ! It's the default value but it's also explicitly set in pyproject.toml, which black scans automatically.

> Oh also, it should be --line-length.

Oh right, nice catch !

Reply:
Oh wait I see your point... I'll remove the argument completely here.